packages/plugin-eslint/src/lib/runner/index.ts

```diff
@@ -23,14 +29,32 @@ export function createRunnerFunction(options: {
   };
 
   return async (): Promise<AuditOutputs> => {
-    logger.info(`ESLint plugin executing ${targets.length} lint targets`);
+    logger.info(
+      `ESLint plugin executing ${pluralizeToken('lint target', targets.length)}`,
+    );
 
     const linterOutputs = artifacts
       ? await loadArtifacts(artifacts)
       : await asyncSequential(targets, lint);
+
     const lintResults = mergeLinterOutputs(linterOutputs);
     const failedAudits = lintResultsToAudits(lintResults);
 
+    const stats = aggregateLintResultsStats(lintResults.results);
+    logger.info(
+      stats.problemsCount === 0
+        ? 'ESLint did not find any problems'
+        : `ESLint found ${pluralizeToken('problem', stats.problemsCount)} from ${pluralizeToken('rule', stats.failedRulesCount)} across ${pluralizeToken('file', stats.failedFilesCount)}`,
+    );
+
+    const totalCount = config.slugs.length;
+    const failedCount = failedAudits.length;
+    const passedCount = totalCount - failedCount;
+    const percentage = roundDecimals((passedCount / totalCount) * 100, 2);
+    logger.info(
+      `${pluralizeToken('audit', passedCount)} passed, ${pluralizeToken('audit', failedCount)} failed (${percentage}% success)`,
+    );
+
     return config.slugs.map(
       (slug): AuditOutput =>
         failedAudits.find(audit => audit.slug === slug) ?? {
```

Comment:
Let's avoid having logic in ìndex.ts`files. This helps us to exclude all ìndex.ts` files from code coverage. This is also mentioned here #1168 under chapter "Test Coverage"